src/arguments.js

```diff
@@ -7,23 +10,39 @@
  * @return {Object} - argument template
  */
 const getArgumentObject = (object, context) => {
-    let argument = context.get.template.argument();
-    argument = {
+    let argument = {
         enum: getEnum(object.arg + object.description, context),
-        longName: getPropertyName(object.arg, context.regexp.argument.long),
-        shortName: getPropertyName(object.arg, context.regexp.argument.short),
+        longNames: getPropertyNames(object.arg, context.regexp.argument.long),
+        shortNames: getPropertyNames(object.arg, context.regexp.argument.short),
         defaultValue: getDefaultValue(
             object.description, context.regexp.argument.default),
         description: unifyDescription(object.description.trim()),
     };
 
+    argument.alias = getAlias(argument);
     argument.usage = !getValueByRegexp(
         argument.description, context.regexp.argument.usage);
     argument.flag = identifyIsFlag(object.arg, argument);
     argument.type = getPropertyType(object.arg, argument, context);
     return argument;
 };
 
+/**
+ * Get alias from names of an argument
+ * @param {object} argument - argument object with description, names, etc.
+ * @return {Array} alias - alias of argument
+ */
+const getAlias = (argument) => {
+    let alias = [];
+    let longNames = argument.longNames;
+    let shortNames = argument.shortNames;
+    if (longNames && longNames[1]) alias.push(longNames.slice(1));
```

Comment:
Many `if` in this function, it's need to improve

Reply:
@svoboda-rabstvo yes, but it is really needed there. Maybe, you have another solution?